### PR TITLE
Some refactorings

### PR DESCRIPTION
**Pull request by janvanmansum**

### CODE REVIEW

**Comment by janvanmansum** [comment]
@PaulBoon @ekoi @GeorgiKhomeriki @jo-pol @vesaakerman please have a look.

File: src/main/scala/nl/knaw/dans/easy/stage/Conf.scala [hunk 1]
```diff
@@ -7,14 +7,15 @@ import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
 
 class Conf(args: Seq[String]) extends ScallopConf(args) {
   printedName = "easy-stage-dataset"
-  version(s"$printedName ${Version()}")
+  val indent_____ = printedName.replaceAll(".", " ")
```
Comment (gkhom) [inline src/main/scala/nl/knaw/dans/easy/stage/Conf.scala:10]: Why the strange variable name with the underscores?

Optionally you could also replace the `printedName.replaceAll(".", " ")` with: `printedName.map(_ => ' ')`. 
I had to think twice about what `replaceAll(".", " ")` did (forgot that the first parameter was a pattern).

Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/stage/Conf.scala:10]: The underscores ensure that the indenting below looks nice both in code and when using the `--help` switch on the command line.

File: src/main/scala/nl/knaw/dans/easy/stage/Conf.scala [hunk 2]
```diff
@@ -7,14 +7,15 @@ import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
...
+  version(s"$printedName v${Version()}")
   banner(s"""
            |Stage a dataset in EASY-BagIt format for ingest into an EASY Fedora Commons 3.x Repository.
            |
            |Usage:
            |
            | $printedName -t <submission-timestamp> -u <urn> -d <doi> [ -o ] \\
-           |                       <EASY-bag> <staged-digital-object-set>
+           | $indent_____    <EASY-bag> <staged-digital-object-set>
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/stage/Conf.scala:18]: ... like this ... :)

File: src/main/scala/nl/knaw/dans/easy/stage/Conf.scala (hunk 1 above)
Comment (gkhom) [inline src/main/scala/nl/knaw/dans/easy/stage/Conf.scala:10]: Makes sense :)

**Pull request merged**